src/integrationTest/java/uk/gov/hmcts/darts/usermanagement/controller/PostSecurityGroupIntTest.java

```diff
@@ -108,6 +108,50 @@ void createSecurityGroupShouldSucceedWhenProvidedWithValidValuesForMinRequiredFi
             assertFalse(createdSecurityGroupEntity.getGlobalAccess());
             assertTrue(createdSecurityGroupEntity.getDisplayState());
             assertEquals("TRANSCRIBER", createdSecurityGroupEntity.getSecurityRoleEntity().getRoleName());
+            assertFalse(createdSecurityGroupEntity.getUseInterpreter());
+
+            return null;
+        });
+    }
+
+    @Test
+    void createSecurityGroupShouldSucceedAndSetInterpreterFlagTrueForTranslationRole() throws Exception {
+        superAdminUserStub.givenUserIsAuthorised(userIdentity);
+
+        MockHttpServletRequestBuilder request = buildRequest()
+            .content("""
+                         {
+                           "name": "ACME",
+                           "display_name": "ACME Transcription Services",
+                           "security_role_id": 5
+                         }
+                           """);
+
+        MvcResult result = mockMvc.perform(request)
+            .andExpect(status().isCreated())
+            .andExpect(jsonPath("$.id").exists())
+            .andExpect(jsonPath("$.name").value("ACME"))
+            .andExpect(jsonPath("$.display_name").value("ACME Transcription Services"))
+            .andExpect(jsonPath("$.description").doesNotExist())
+            .andExpect(jsonPath("$.display_state").value(true))
+            .andExpect(jsonPath("$.global_access").value(false))
+            .andExpect(jsonPath("$.security_role_id").isNumber())
+            .andReturn();
+
+        var id = new JSONObject(result.getResponse().getContentAsString())
+            .getInt("id");
+
+        transactionTemplate.execute(status -> {
```

Comment:
For these asserts that need to be transactional, I created a utility that does the same thing really but slightly neater i think.  TransactionalAssert.  Up to you if you want to use it, just a bit of syntactic sugar.

Reply:
Nice one, I'll take a look and will update.